compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java

```diff
@@ -342,6 +342,7 @@ public CodeAnalyzer(CompilerContext context) {
 
     public BLangPackage analyze(BLangPackage pkgNode) {
         this.workerReferences.clear();
+        this.reachabilityAnalyzer.clearStacks();
```

Comment:
What about doing this immediately before https://github.com/ballerina-platform/ballerina-lang/blob/stage-swan-lake/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/CodeAnalyzer.java#L536 instead?